workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -22,6 +22,8 @@ internal static class GenerateCode
         private static readonly string StartupCodegenMarkerFile =
             Path.GetFullPath(Path.Combine("Temp", "ImprobableCodegen.marker"));
 
+        private static string SchemaCompilingErrorMessage = "Error(s) compiling schema files!";
```

Comment:
This shouldn't be static, its being used as a global, mutable variable. 

Instead we can make this local and pass in a lambda to `AddErrorProcessing` rather than a static function.

I.e. - 

```csharp
// Inside Generate()
var errorMessage = new StringBuilder();

var exitCode = RedirectedProcess.Command(Common.DotNetBinary)
	...
	.AddErrorProcessing((line) => errorMessage.Add(line))
	...
	.Run();

Debug.LogError($"Error(s) compiling schema files!\n{errorMessage.ToString()});
```